src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+const man = 'm';
+const woman = 'f';
```

Comment:
```suggestion
const MAN = 'm';
const WOMAN = 'f';
```
we usually name constants in uppercase